src/javascript/DesignSystem/ContentTable/ContentTable.jsx

```diff
@@ -27,10 +27,37 @@ const styles = theme => ({
         '&:hover': {
             backgroundColor: theme.palette.background.default
         }
+    },
+    selectedRow: {
+        '&&&': {
```

Comment:
Why 3 & ?

I don't know this selector

Reply:
This came from the CMM table styles, in any case the table will be redesign

Reply:
oki :)